docs/source/user/jit.rst

```diff
@@ -183,6 +183,25 @@ a file-based cache.  This is done by passing ``cache=True``::
    def f(x, y):
        return x + y
 
+
+
+.. note::
+    Caching of compiled functions has several known limitations:
+
+    - The caching of compiled functions is not performed on a
+      function-by-function basis. The cached function is the the main jit
+      function, and all secondary functions (those called by the main
+      function) are incorporated in the cache of the main function.
+    - Cache invalidation fails to recognize changes in symbols defined in a
```

Comment:
```suggestion
    - Cache invalidation fails to recognize changes in functions defined in a
```
whilst `symbols` is probably more correct given what goes on, as this are user facing documentation, it might not be known to users what actually happens when code is compiled and the associated terminology, so I wonder if `functions` better serves this purpose?